llvm/test/CodeGen/AMDGPU/abi-attribute-hints-undefined-behavior.ll

```diff
@@ -255,7 +255,7 @@ define amdgpu_kernel void @marked_kernel_use_other_sgpr(ptr addrspace(1) %ptr) #
   %queue.load = load volatile i8, ptr addrspace(4) %queue.ptr
   %implicitarg.load = load volatile i8, ptr addrspace(4) %implicitarg.ptr
   %dispatch.load = load volatile i8, ptr addrspace(4) %dispatch.ptr
-  store volatile i64 %dispatch.id, ptr addrspace(1) %ptr
+  store i64 %dispatch.id, ptr addrspace(1) %ptr
```

Comment:
Why did this lose a volatile?

Reply:
By dropping the volatile we preserve the original FileCheck pattern.  I'm happy to restore the volatile and update the pattern match instead, that's probably the better choice given the intent of this patch.

Reply:
It's supposed to just be a dummy use. You could equally just remove the store, I guess